### PR TITLE
[v2] Rename node __id arg to id

### PR DESCRIPTION
**Pull request by alloy**
Closes https://artsyproduct.atlassian.net/browse/PLATFORM-1441

### CODE REVIEW

File: patches/graphql-tools+4.0.4.patch [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+--- a/node_modules/graphql-tools/dist/stitching/resolvers.js
++++ b/node_modules/graphql-tools/dist/stitching/resolvers.js
+@@ -51,7 +51,9 @@ function createProxyingResolver(schema, operation, fieldName, transforms) {
+         schema: schema,
+         operation: operation,
+         fieldName: fieldName,
+-        args: {},
++        // It is unclear if passing on `args` here is without downsides or this
++        // _has_ to be `{}`.
++        args: args,
```
Comment (alloy) [inline patches/graphql-tools+4.0.4.patch]: Need to test if this change breaks tests in graphql-tools.

Comment (alloy) [reply patches/graphql-tools+4.0.4.patch]: Does not break any tests 👍

**Comment by alloy** [comment]
Also made the test output clean again, which involved replacing deprecated `Buffer` usage, so do look at that closely.

File: src/lib/cache.ts [hunk 2]
```diff
@@ -111,7 +111,7 @@ function _get<T>(key) {
         if (CACHE_COMPRESSION_DISABLED) {
           resolve(JSON.parse(data.toString()))
         } else {
-          zlib.inflate(new Buffer(data, "base64"), (er, inflatedData) => {
+          zlib.inflate(Buffer.from(data, "base64"), (er, inflatedData) => {
```
Comment (zephraph) [inline src/lib/cache.ts]: 👍

File: src/schema/sale/__tests__/index.test.js [hunk 3]
```diff
@@ -175,7 +175,7 @@ describe("Sale type", () => {
     })
   })
 
-  describe("sale_artworks_connection", async () => {
+  describe("sale_artworks_connection", () => {
```
Comment (zephraph) [inline src/schema/sale/__tests__/index.test.js]: I fixed some of these yesterday in reaction. I think we should find/write a lint rule to catch these.

Comment (alloy) [reply src/schema/sale/__tests__/index.test.js]: Perhaps, yeah

**Comment by zephraph** [comment]
I did some local testing on this and it looks good! Gonna merge. 

Thanks man!

**Pull request merged**